conductr_cli/validation.py

```diff
@@ -219,11 +219,7 @@ def handler(*args, **kwargs):
             log = get_logger_for_func(func)
             log.error('Docker native is installed and Docker machine environment variables are set.')
             log.error('It is uncertain which Docker VM should be used.')
-            log.error('If Docker native should be used please unset the Docker machine environment variables:')
-            log.error('  DOCKER_CERT_PATH')
-            log.error('  DOCKER_HOST')
-            log.error('  DOCKER_MACHINE_NAME')
-            log.error('  DOCKER_TLS_VERIFY')
+            log.error('If Docker native should be used please unset the DOCKER_MACHINE_NAME environment variable.')
```

Comment:
This advice can lead to misfunctioning Docker.

**Example**
- The user is setting the Docker machine VM variables with:

```
eval "$(docker-machine env default)"
```
- Now the user has also started Docker native. As a result, he is receiving the error message above
- Unsetting only `DOCKER_MACHINE_NAME` will not work. The native docker daemon will not be able to connect to the native docker host because `DOCKER_HOST` points to the VirtualBox VM Docker host. In this case, the user need to unset all 4 environment variables.